packages/odata-common/src/entity-base.ts

```diff
@@ -351,7 +351,7 @@ export function entityBuilder<
   Object.values(entityApi.schema).forEach((field: any) => {
     const fieldName = `${camelCase(field._fieldName)}`;
     builder[fieldName] = function (value) {
-      this.entity[fieldName] = value;
+      this._entity[fieldName] = value;
```

Comment:
Yes it will fail if there is a property `_entity`. The reserved words list already has `Entity` but I believe that has no effect on naming of property fields.

Reply:
@FrankEssenberger I added both `entity` and `ENTITY` to the `reservedVdmKeywords` for instance and static property naming. So entity will now be formatted to instance property `entity_1` and static `ENTITY_1` in schema.

Reply:
I already approved, I had the question why you renamedit with the `_`?